apps/vscode/vscode.talon

```diff
@@ -144,11 +153,12 @@ git stash pop: user.vscode("git.stashPop")
 git status: user.vscode("workbench.scm.focus")
 git stage: user.vscode("git.stage")
 git stage all: user.vscode("git.stageAll")
+git sync: user.vscode("git.sync")
 git unstage: user.vscode("git.unstage")
 git unstage all: user.vscode("git.unstageAll")
 pull request: user.vscode("pr.create")
-change next: key(alt-f5)
-change last: key(shift-alt-f5)
+change next: user.vscode("workbench.action.compareEditor.nextChange")
+change last: user.vscode("workbench.action.compareEditor.previousChange")
```

Comment:
Unfortunately, this will break `change next` when you're in a normal view, (ie not in a compare editor).  VSCode relies on when clause contexts to determine which one to use:

```json
{
  "key": "alt+f5",
  "command": "workbench.action.compareEditor.nextChange",
  "when": "textCompareEditorVisible"
},
{
  "key": "alt+f5",
  "command": "workbench.action.editor.nextChange",
  "when": "editorTextFocus && !textCompareEditorActive"
}
```

And of course, we cannot access these contexts (see https://github.com/microsoft/vscode/pull/118722 and https://github.com/microsoft/vscode/issues/117980).  So we need to use the keyboard shortcut if we want to do the right thing in both contexts.  I think the solution for cross-platform support here is to define actions

Reply:
Done